packages/aws-cdk-lib/aws-apigatewayv2/README.md

```diff
@@ -93,6 +93,15 @@ new apigwv2.HttpApi(this, 'HttpProxyApi', {
 });
 ```
 
+The `routeSelectionExpression` option can be configured.
+In the HTTP API, only the value `$request.method $request.path` is allowed, and it can be configured by enabling `routeSelectionExpression`.
```

Comment:
There is a bit of redundancy in the first and second halves, could you simplify it a bit?